tests/output/sapi_windows_vt100_support_non_castable_user_stream.phpt

```diff
@@ -0,0 +1,88 @@
+--TEST--
+sapi_windows_vt100_support(): uncastable user stream
+--SKIPIF--
+<?php
+if(PHP_OS_FAMILY !== "Windows") {
+    echo "skip Only for Windows systems";
+} elseif (version_compare(
+    PHP_WINDOWS_VERSION_MAJOR.'.'.PHP_WINDOWS_VERSION_MINOR.'.'.PHP_WINDOWS_VERSION_BUILD,
+     '10.0.10586'
+) >= 0) {
+    echo "skip Only for Windows systems < 10.0.10586";
+}
+?>
+--FILE--
+<?php
+class DummyStreamWrapper
```

Comment:
Wouldn't be better to separate this class and just include it here and elsewhere so it is not repeated...?